docs/challenges/language-support/go.mdx

```diff
@@ -52,9 +55,10 @@ Let's say you wanted to extract `foo.go` into the `bar` package. You'd move `foo
 package bar
 
 fun foo() {
-    println("Hello world!")
+println("Hello world!")
```

Comment:
@andy1li this diff seems wrong, it removes indentation

Reply:
Hmm, let me find out why it happened and re-check all the other diffs.